lib/mongoose_push/application.ex

```diff
@@ -86,6 +88,39 @@ defmodule MongoosePush.Application do
     end)
   end
 
+  defp metrics do
```

Comment:
I feel like this should belong in some separate module dedicated for that. This will only get bigger.